secure-storage/secure-storage-api/src/main/java/com/duckduckgo/securestorage/api/SecureStorage.kt

```diff
@@ -93,6 +93,19 @@ interface SecureStorage {
     @Throws(SecureStorageException::class)
     suspend fun websiteLoginDetailsWithCredentials(): Flow<List<WebsiteLoginDetailsWithCredentials>>
 
+    /**
+     * This method returns all the [WebsiteLoginDetailsWithCredentials] stored in the [SecureStorage] modified [since].
+     * This returns decrypted sensitive data (encrypted in L2). Use this only when sensitive data is needed to be accessed.
+     * If [canAccessSecureStorage] is false when this is invoked, an empty flow will be emitted.
```

Comment:
> If [canAccessSecureStorage] is false when this is invoked, an empty flow will be emitted.

if this does happen, what will sync do with that?

Reply:
Since we have timestamp in metadata table, we don't need to use this method anymore 🙌